lib/Ray/sources/Audio/IAudio.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Audio
+*/
+
+#ifndef AUDIO_HPP_
+#define AUDIO_HPP_
+
+#include "IRessource.hpp"
+#include <string>
+
+namespace RAY::Audio
+{
+	//! @brief Interface for Audio ressources
+	class IAudio: public IRessource {
+		public:
+			virtual ~IAudio() = 0;
```

Comment:
A destructor should not be disabled.

Reply:
fixed -> https://github.com/AnonymusRaccoon/Bomberman/pull/40/commits/3d5ce0ff8142cbde4da41ea9aa8d8862279efc7c